config/initializers/feature_flags.rb

```diff
@@ -0,0 +1,11 @@
+FEATURES = {
+  basic_auth: {
+    enabled:  Rails.env.production? || (!ENV['BASIC_AUTH_FORCE'].blank? && ENV['BASIC_AUTH_FORCE'] != 'false'),
+    username: ENV['BASIC_AUTH_USERNAME'],
+    password: ENV['BASIC_AUTH_PASSWORD'],
+  },
+  support_auth: {
+    username: ENV['SUPPORT_USERNAME'],
+    password: ENV['SUPPORT_PASSWORD'],
+  },
+}.to_hash.freeze # rubocop does not detect freezing without the to_hash
```

Comment:
But... this is a hash literal! What's up with Rubocop?

Reply:
Turns out we are on the latest version of rubocop. I guess rubocop gets confused by the dynamic nature of the hash contents. Moving hash initialisation to a temp var, then setting and freezing the constant makes rubocop happy.